code/datums/wires/_wires.dm

```diff
@@ -145,6 +148,7 @@
 	if(is_cut(wire))
 		return
 	on_pulse(wire, user)
+	feedback(wire)

```

Comment:
Couldn't produce in testing but this has me nervous that receiving a pulse from a signaler will issue a pulse back to that signaler somehow.

Reply:
I'll see if i can make it ignore pulses from attached assemblies